yogstation/code/modules/donor/unique_donator_items.dm

```diff
@@ -0,0 +1,542 @@
+/**
+
+This file contains any "special" / unique donator items which are ckey locked. 
+
+To create a new unique donator item, create a new datum/donator_item
+
+@version 1.0
+@author Kmc2000
+
+*/
+
+GLOBAL_LIST_INIT(donor_pdas, list("Normal", "Transparent", "Pip Boy", "Rainbow"))
+
+GLOBAL_DATUM_INIT(donator_gear, /datum/donator_gear_resources, new)
+
+/client/proc/custom_donator_item()
+	if(!is_donator(src))
+		to_chat(src, "<span class='warning'>You're not a donator! To access this feature, considering donating today!</span>")
+		return
+	GLOB.donator_gear.ui_interact(usr)//datum has a tgui component, here we open the window
+
+/datum/donator_gear_resources
+	var/name = "Unique Donator Items Controller"
+	var/list/donor_items = list()
```

Comment:
```suggestion
	var/name = "Unique Donator Items Controller"
	///[description of what donor_items is and what it will contain]
	var/list/donor_items = list()
```
Also. each datum below should be documented similar to the proc above